tests/phpunit/api/v3/ContributionPageTest.php

```diff
@@ -1847,27 +1844,26 @@ public function testSubmitPledgePayment(): void {
       'pledge_id' => $pledge['id'],
       'cid' => $pledge['contact_id'],
       'contact_id' => $pledge['contact_id'],
-      'amount' => 100.00,
       'is_pledge' => TRUE,
       'pledge_block_id' => $this->_ids['pledge_block_id'],
     ];
     $pledgePayment = $this->callAPISuccess('pledge_payment', 'get', $params);
     $this->assertEquals(2, $pledgePayment['values'][2]['status_id']);
 
-    $this->callAPIAndDocument('contribution_page', 'submit', $submitParams, __FUNCTION__, __FILE__, 'submit contribution page', NULL);
+    $this->callAPIAndDocument('contribution_page', 'submit', $submitParams, __FUNCTION__, __FILE__, 'submit contribution page');
 
     // Check if contribution created.
-    $contribution = $this->callAPISuccess('contribution', 'getsingle', [
+    $contribution = $this->callAPISuccess('Contribution', 'getsingle', [
       'contribution_page_id' => $pledge['pledge_contribution_page_id'],
       'contribution_status_id' => 'Completed',
       'contact_id' => $pledge['contact_id'],
       'contribution_recur_id' => ['IS NULL' => 1],
     ]);
 
-    $this->assertEquals(100.00, $contribution['total_amount']);
-    $pledgePayment = $this->callAPISuccess('pledge_payment', 'get', $params)['values'];
+    $this->assertEquals(10.00, $contribution['total_amount']);
+    $pledgePayment = $this->callAPISuccess('PledgePayment', 'get', $params)['values'];
     $this->assertEquals(1, $pledgePayment[2]['status_id'], 'This pledge payment should have been completed');
-    $this->assertEquals($contribution['id'], $pledgePayment[2]['contribution_id']);
+    $this->assertEquals($pledgePayment[2]['contribution_id'], $contribution['id']);
```

Comment:
@eileenmcnaughton wasn't this around the right way before?

Reply:
Yeah maybe - can I fix that as a follow up?

Reply:
as long as you promise to do so

Reply:
I've added it into https://github.com/civicrm/civicrm-core/pull/20402